src/applications/mhv-landing-page/components/CardLayout.jsx

```diff
@@ -50,7 +50,7 @@ const CardLayout = ({ data }) => {
             key={`col-${y}`}
           >
             {col.title === HEALTH_TOOL_HEADINGS.MEDICAL_RECORDS &&
-            mhvTransitionalMedicalRecordsLandingPage ? (
+            !mhvIntegrationMedicalRecordsToPhase1 ? (
```

Comment:
Note: I cannot remove the transitional toggle from the entire `vets-website` repo yet as it is still used. Will be completely removed in another ticket.